### ISSUE TITLE
Not being used by ARR

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-style-files#37]
It seems that [the ARR template in Overleaf](https://www.overleaf.com/latex/templates/acl-rolling-review-template/jxbhdzhmcpdm) doesn't use this. Anyone know who's in charge, that we could fix this?

**Comment by LSinev** [comment]
Maybe this will help https://github.com/acl-org/acl-style-files/issues/6#issuecomment-1771419810

**Comment by mjpost** [comment]
Thanks—that's a separate Overleaf, unfortunately. But it's a good idea to see if Dan knows about the ARR one.

**Comment by davidweichiang** [comment]
My very dim recollection is that Amanda Stent may have uploaded it, and once it's uploaded as a template, it's managed by Overleaf, not a user. For that reason she (or whoever it was that did it) recommended not doing this again.

**Comment by davidweichiang** [comment]
Is that template actually linked from somewhere, or do you find it by searching on Overleaf?

related: https://github.com/acl-org/aclrollingreview/issues/41

**Comment by mjpost** [comment]
It was not in fact linked from ARR—I only found it by googling. If it can't be updated, we should remove it, and switch over to the ACL template.

**Comment by mjpost** [comment]
It seems however that one [*can update a template](https://www.overleaf.com/learn/how-to/Publishing_templates_and_examples_in_the_Overleaf_Gallery#How_to_update_a_published_template). So we should publish the ACL one as a template in the Overleaf gallery, and then manually update that every time it changes.

**Comment by davidweichiang** [comment]
Ah, here's the old discussion: https://github.com/acl-org/acl-style-files/issues/6#issuecomment-1003428426

**Comment by mjpost** [comment]
So that issue discusses syncing the Github repo with Overleaf. Publishing it as a template in the Overleaf gallery is a separate step.

I think the steps therefore are:
- [ ] Find the owner of the Overleaf project that produced the ARR template, and delete it
- [ ] Ensure there are no real important differences
- [ ] Publish ours instead as the official ACL template, possibly with a name like "ACL/ARR Template" to reduce confusion

**Comment by mjpost** [comment]
Ah, I see, I just jumped to the top of the thread and missed the particular comment you linked to.

I still think we might pursue the path of using an official template. We have an official project, so once this is straightened out, it shouldn't need unpublishing, just updating. Do we have access to the ARR template (@ajstent?) that we could test updating it and seeing if it gets through moderation?

Regardless we should try to get that one deleted, since it is a point of confusion even for an insider like me. ARR should also link to whatever instructions we come up with.

**Comment by mjpost** [comment]
ARR [links to the Github repo](https://aclrollingreview.org/cfp#paper-submission-and-templates); is there a read-only public link to the Overleaf that they could additionally link to? That already would be a lot more convenient than the Github repo. I checked, but am unable to do anything since I am no longer the owner.

**Comment by mjpost** [comment]
And now I see that we link there in the Github README: https://www.overleaf.com/read/crtcwgxzjskr. 

So maybe the simplest path forward is to try to remove the ARR template, but this seems less pressing since there is a clear path for people who read the instructions.

**Comment by davidweichiang** [comment]
> And now I see that we link there in the Github README: https://www.overleaf.com/read/crtcwgxzjskr.

I very dimly remember that to avoid creating O(n^2) links, I asked for everyone to link to the Github README and the Github README links everywhere else.

**Comment by davidweichiang** [comment]
> Here are a few more weird things about the current method of sharing via a non-template

Agreed on all of these. I imagine that even non-editors would be worried that they are editing the template for everyone else.

**Comment by neubig** [comment]
It wasn't me AFAIR

**Comment by mjpost** [comment]
Okay, I wrote to Overleaf to see if they'll remove [the ARR template](https://www.overleaf.com/latex/templates/acl-rolling-review-template/jxbhdzhmcpdm).

**Comment by mjpost** [comment]
It's done! They deleted it. 

Their expeditiousness causes me to think things have changed enough from two years ago that we might successfully publish and link to an official template from this repo.

**Comment by mjpost** [comment]
@danielgildea What do you think about publishing an official template, then, and linking to that from the README? As [I noted in a comment above](https://github.com/acl-org/acl-style-files/issues/37#issuecomment-1946531250), there's a lot of awkwardness in actually instantiating the current project on Overleaf, and Overleaf's responsiveness in deleting the old ARR template suggests maybe the customer support issues from a few years ago have been resolved.

**Comment by davidweichiang** [comment]
It's great that they were so responsive! I wonder if it makes sense for someone (@mjpost?) to communicate with them further about (a) whether they can stay perpetually synced with a Github repository or (b) if they have to do manual review, whether they have any expectations about what a reasonable frequency of updates is and whether they can make any commitment to a maximum turnaround time.

**Comment by mjpost** [comment]
Good thought, I'll use my open thread with them to ask.

**Comment by danielgildea** [comment]
> @danielgildea What do you think about publishing an official template, then, and linking to that from the README? 

I can do that, if you are happy with the answers from overleaf to your questions.

**Comment by mjpost** [comment]
I just remembered this. I emailed last week and got a response right away. All initial approvals and changes do require manual review. They expect that templates are pretty stable. I didn't ask about whether we can keep permanently synced between Github and Overleaf but that has been a feature for some time. She did mention that changes to the template are easier if they are issued from the same Overleaf repo.

Based on this, I suggest we publish a template and then update our internal link(s) to point to it.
